var/spack/repos/builtin/packages/py-atlas-building-tools/package.py

```diff
@@ -38,7 +39,7 @@ class PyAtlasBuildingTools(PythonPackage):
     depends_on('py-scikit-image@0.17.2:', type=('build', 'run'))
     depends_on('py-tqdm@4.44.1:', type=('build', 'run'))
     depends_on('py-trimesh@2.38.10:', type=('build', 'run'))
-    depends_on('py-voxcell@3.0.0', type=('run', 'build'))
+    depends_on('py-voxcell@3.0.0:', type=('run', 'build'))
```

Comment:
Out of curiosity, is the `build` type required in these dependencies? I was looking at the brainbuilder for example and it seems to specify all dependencies using the `run` type only:
https://github.com/BlueBrain/spack/blob/develop/var/spack/repos/builtin/packages/brainbuilder/package.py

Reply:
See here: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#python-dependencies, as noted by @GianlucaFicarelli in #1337 

Seems like that should actually be `build` and `run` to keep `pip` from downloading extra stuff.